### PR TITLE
Update Multiple Resolutions demo for Godot 4.0

### PR DESCRIPTION
**Pull request by Calinou**
See https://github.com/godotengine/godot-demo-projects/issues/697.

## Preview

image

<details>
<summary>Older revision</summary>

multiple_resolutions
</details>

### CODE REVIEW

File: gui/multiple_resolutions/README.md [hunk 1]
```diff
@@ -39,13 +39,13 @@ factor is changed).
 
 Language: GDScript
 
-Renderer: GLES 2
+Renderer: Vulkan
```
Comment (aaronfranke) [inline gui/multiple_resolutions/README.md:42]: We should specify Vulkan Clustered or Vulkan Mobile. For this demo probably Vulkan Mobile.

**Comment by Calinou** [comment]
- Changed rendering backend to Vulkan Mobile.
- Enabled MSDF for the default font to allow for faster resizing of the window. The font no longer needs to be re-rasterized when its size changes.
- Remove unused font file (we use the default font now).

**Comment by aaronfranke** [comment]
I'm not sure what the purpose of the side labels is, but it overlaps with the text. Maybe we should remove the side labels?

Screenshot from 2022-05-07 17-10-53

**Comment by Calinou** [comment]
@aaronfranke Done, I moved the labels to the top-left and bottom-left corners.

**Review by aaronfranke — APPROVED** [review]

**Pull request merged**